src/program/lwaftr/generate_binding_table/README

```diff
@@ -0,0 +1,14 @@
+Usage:
+
+generate_binding_table <ipv4> <num_ipv4s> <br_address> <b4> <psid_len> <shift>
+    --output <filename>: Write down binding-table to file.
+
```

Comment:
That has 6 arguments; the examples have 5. I assume that 'shift' is the undocumented one?